guides/common/modules/con_managing-organizations.adoc

```diff
@@ -7,7 +7,9 @@ Organizations divide {ProjectName} resources into logical groups based on owners
 ifdef::satellite[]
 You can create and manage multiple organizations through {ProjectName}, then divide and assign your Red Hat subscriptions to each individual organization.
 endif::[]
-ifndef::satellite[]
+ifdef::katello,orcharhino[]
```

Comment:
Katello doesn't allow nested organizations while vanilla Foreman does
```suggestion
ifndef::katello,orcharhino,satellite[]
```

Reply:
This would make it foreman-el & foreman-deb only, but it's about RH subscriptions which is Katello-only, isn't it? -> Should I drop line 10-12 then?

Reply:
I merged it the macros; it's now hidden for non-Katello builds.